tasty/src/dotty/tools/tasty/TastyFormat.scala

```diff
@@ -265,8 +265,15 @@ All elements of a position section are serialized as Ints
 
 Standard Section: "Comments" Comment*
 ```none
-  Comment       = Length Bytes LongInt      // Raw comment's bytes encoded as UTF-8, followed by the comment's coordinates.
+  Comment       = UTF8 LongInt              // Raw comment's bytes encoded as UTF-8, followed by the comment's coordinates.
 ```
+
+Standard Section: "Attributes" Attribute*
+```none
+  Attribute     = SCALA2STANDARDLIBRARYattr
+                  EXPLICITNULLSattr
```

Comment:
I was wondering if this is the correct way to define experimental language attributes. We might end up at some point with attributes that are never used in stable code. On the other hand, we make sure that all attributes that we ever use are unambiguous.

Reply:
Explicit-nulls is not experimental, at least not in the meaning of `@experimental`. For historical reasons, it benefits from a favored treatment and is allowed in stable, non-experimental builds (it only needs the flag).